packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -175,34 +175,46 @@ class Transforms {
 
     const promises = [];
 
-    return ctx.webex.internal.ediscovery.getContentSummaryBySpaceId(reportId, activity.targetId)
+    return ctx.webex.internal.ediscovery.getContentContainerByContainerId(reportId, activity.targetId)
```

Comment:
Switch over to the new `/container` API rather than call the, now empty initially, space summary cache.